### ISSUE TITLE
Patellar overgrowth

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6419]
**Preferred term label:**
Patellar overgrowth

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://pubmed.ncbi.nlm.nih.gov/12928894/   2/3
https://omim.org/entry/607115

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
patellar hypertrophy, synonym

https://pubmed.ncbi.nlm.nih.gov/12483741/   6/6

**Comment by pnrobinson** [comment]
added
